src/content/docs/en/guides/integrations-guide/image.mdx

```diff
@@ -21,9 +21,9 @@ import DontEditWarning from '~/components/DontEditWarning.astro';
 
 <DontEditWarning/>
 
-> ⚠️ This integration is still experimental! Only node environments are supported currently, stay tuned for Deno support in the future!
+> ⚠️ This integration will be deprecated in Astro v3.0 (Fall 2023) in favor of the `astro:assets` module. Please see the [Assets documentation](/en/guide/assets/) for more information.
```

Comment:
We'll need to fix this upstream in the image integration README:

```suggestion
> ⚠️ This integration will be deprecated in Astro v3.0 (Fall 2023) in favor of the `astro:assets` module. Please see the [Assets documentation](/en/guides/assets/) for more information.
```

Reply:
Gah! Will fix this now

Reply:
Fixed upstream: PR here: https://github.com/withastro/astro/pull/7649